test/spec_utils.rb

```diff
@@ -702,17 +658,10 @@ def initialize(*)
     Rack::Utils.delete_cookie_header!(header, 'name').must_be_nil
     header['set-cookie'].must_equal "name=; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 GMT"
   end
-
-  deprecated "sets deleted cookie" do
-    Rack::Utils.make_delete_cookie_header(nil, 'name', {}).
-      must_equal "name=; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 GMT"
-    Rack::Utils.add_remove_cookie_to_header(nil, 'name').
-      must_equal "name=; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 GMT"
-  end
 end
 
 describe Rack::Utils, "get_byte_ranges" do
-  deprecated "pase simple byte ranges from env" do
+  it "pase simple byte ranges from env" do
```

Comment:
Not a bug in the PR, but may as well fix the typo in the description: `s/pase/parse/`

Reply:
```suggestion
  it "parse simple byte ranges from env" do
```